### PR TITLE
fixes #2

### LINKED ISSUE #2: Add maven flow
Add maven flow in GitHub Actions

### PR DESCRIPTION
**Pull request by cichan02**

### CODE REVIEW

File: src/test/java/by/piskunou/solvdlaba/service/AirlineServiceTest.java [hunk 1]
```diff
@@ -15,8 +15,7 @@
 
 import java.util.Optional;
 
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (h1alexbel) [inline src/test/java/by/piskunou/solvdlaba/service/AirlineServiceTest.java:18]: don't use *

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: src/test/java/by/piskunou/solvdlaba/service/AirlineServiceTest.java (hunk 1 above)
Comment (h1alexbel) [reply src/test/java/by/piskunou/solvdlaba/service/AirlineServiceTest.java:18]: @cichan02 [this](https://stackoverflow.com/questions/26225989/how-do-i-stop-wildcard-java-package-imports-in-intellij) can be helpful

File: src/test/java/by/piskunou/solvdlaba/service/AirplaneServiceTest.java [hunk 2]
```diff
@@ -70,13 +73,10 @@ void verifyCreateTest() {
...
-            @Override
-            public Void answer(InvocationOnMock invocation) throws Throwable {
-                Airplane airplane = invocation.getArgument(0);
-                airplane.setId(1L);
-                return null;
-            }
+        doAnswer(invocation -> {
+            Airplane airplane1 = invocation.getArgument(0);
+            airplane1.setId(1L);
+            return null;
```
Comment (h1alexbel) [inline src/test/java/by/piskunou/solvdlaba/service/AirplaneServiceTest.java:79]: Try to don't return ```null```. It's a bad practice

https://www.yegor256.com/2014/05/13/why-null-is-bad.html

Comment (h1alexbel) [inline src/test/java/by/piskunou/solvdlaba/service/AirplaneServiceTest.java:79]: Try to don't return ```null```. It's a bad practice

https://www.yegor256.com/2014/05/13/why-null-is-bad.html

**Review by h1alexbel — APPROVED** [review]

**Review by h1alexbel — APPROVED** [review]

File: src/test/java/by/piskunou/solvdlaba/service/AirplaneServiceTest.java (hunk 2 above)
Comment (cichan02) [reply src/test/java/by/piskunou/solvdlaba/service/AirplaneServiceTest.java:79]: And what to return in this case?

**Pull request merged**